File: Python/cours/008_modularité/includes/player.py
#coding:utf-8

#Module pour le joueur

def parler(personnage, message):
	print("{} : {}".format(personnage, message))

def au_revoir():
	print("Au revoir :) !")


if __name__ == "__main__":
	print("PHASE DE TEST DE PLAYER.PY")

	parler("Jason", "Bonjour tout le monde")
	au_revoir()

File: Java/cours/024_files/Player.java
public class Player
{
    public Player(String name, int level)
    {
        this.name = name;
        this.level = level;
    }

    public void hello()
    {
        System.out.println(this.name + " dit bonjour !");
    }

    /*---------------------------------------------------------*/

    private String name;
    public String getName(){return this.name;}

    private int level;
}